pkg/lint/lintersdb/manager.go

```diff
@@ -506,6 +506,11 @@ func (m Manager) GetAllSupportedLinterConfigs() []*linter.Config {
 			WithLoadForGoAnalysis().
 			WithURL("https://github.com/Antonboom/errname").
 			WithSince("v1.42.0"),
+		linter.NewConfig(golinters.NewContextCheck()).
+			WithPresets(linter.PresetBugs).
```

Comment:
I don't think it's necessarily a bug not to pass the context.
Of course, I know in most cases it is better to pass the context.

Reply:
This linter doesn't check if the context is passed but the heritage of a context.
In a lot of cases, it's a bug.